modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -379,7 +468,6 @@ def process_circuits(
         * `wasm_file_handler`: a ``WasmFileHandler`` object for linked WASM module.
         * `pytketpass`: a ``pytket.passes.BasePass`` intended to be applied
            by the backend (beta feature, may be ignored).
-        * `use_websocket`: boolean flag to allow using a websocket connection.
```

Comment:
Hm, seems this was unused. Yet it is used in the call from the `cost()` function. Is this a bug?

Reply:
It was used when process_circuits was handling batching as it would check on the status of the first job (which is why the retrieve_job calling methods need it). It should have been removed in #450  but was leftover.

Reply:
OK. Will its use by the `cost()` method be fixed in a separate PR?

Reply:
no, that was an oversight, fixed now